src/main/docs/guide/dbc/dbcCriteriaSpecifications/typeSafeJava.adoc

```diff
@@ -19,7 +19,7 @@ Note that as of this writing you cannot use Micronaut Data annotations (those fo
 
 To configure the metamodel generator simply add the following dependency to the annotation processor classpath:
 
-dependency:hibernate-jpamodelgen-jakarta[groupId="org.hibernate", scope="annotationProcessor"]
+dependency:hibernate-jpamodelgen[groupId="org.hibernate.orm", scope="annotationProcessor"]
 
 NOTE: The Hibernate 6 version of `hibernate-jpamodelgen-jakarta` is required because prior versions of Hibernate are still using the `javax.persistence` package
```

Comment:
Please changes also 
```The Hibernate 6 version of `hibernate-jpamodelgen-jakarta```
to
```The Hibernate 6 version of `hibernate-jpamodelgen```

Reply:
@dstepanov @radovanradic I am having problems to commit a new patch following your comments.
I am not able to push another commit, I am rejected. So I synchronize my fork as it says in the next link: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork#syncing-a-fork-branch-from-the-command-line but instead of main I am merging with 4.7.x, I got the next result:

68048a45c (HEAD -> patch-1) Update typeSafeJava.adoc
2c5dd3b447 (upstream/4.7.x) docs: clarify how to add annotation processor dependencies for Kotlin and Groovy projects. (#2910)
2628b21e1b (origin/4.7.x) Rollback PRs intended for 4.8.x (#2889)
5c49c0dd8e Support system/pseudo column as version field (ORA_ROWSCN) (#2873)

but still I am rejected, could you please let me know what I am doing wrong?

Reply:
Don't know what could be wrong, maybe would be easier to close this PR and open a new one against 4.8.x.